lms/djangoapps/grades/events.py

```diff
@@ -215,7 +215,7 @@ def fire_segment_event_on_course_grade_passed_first_time(user_id, course_locator
 
     * Event should be only fired for learners enrolled in paid enrollment modes.
     """
-    event_name = LEARNER_PASSED_COURSE_FIRST_TIME
+    event_name = 'edx.course.learner.passed.first_time'
```

Comment:
any particular reason for changing this?

Reply:
cleanup